services/galley/src/Galley/API/Swagger.hs

```diff
@@ -110,6 +134,29 @@ instance ToSchema NewLegalHoldService where
           }
 
 instance ToSchema ViewLegalHoldService where
+    declareNamedSchema _ = pure $ NamedSchema (Just "ViewLegalHoldService") $ mempty
+        & properties .~ properties_
+        & example .~ example_
+        & required .~ ["status"]
+        & minProperties .~ Just 1
+        & maxProperties .~ Just 2
+        & type_ .~ SwaggerObject
+      where
+        properties_ :: InsOrdHashMap Text (Referenced Schema)
+        properties_ = fromList
+          [ ("status", Inline (toSchema (Proxy @Text)))  -- TODO: this is an enum, but it can't be generated!
```

Comment:
Why can it not be generated?

Reply:
i added this:

```
               -- This is an enum, but it can't be generated, since it is not an instance of
               -- 'Bounded' and 'Enum'.  (TODO: make it an enum in swagger anyway, i'm sure
               -- there is such a thing.  to make sure you're getting it right, define @data
               -- TempStatus = Disabled | NotConf | Conf@ and generate the swagger docs, then
               -- as an exercise create the same docs with swagger2 machinery.)
```